.github/workflows/ci.yml

```diff
@@ -0,0 +1,236 @@
+---
+
+name: PyYAML CI
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  LIBYAML_REPO: https://github.com/yaml/libyaml
+  LIBYAML_REF: '0.2.5'
+jobs:
+  python_sdist:
+    name: pyyaml sdist
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout pyyaml
+        uses: actions/checkout@v2
+
+      - name: install a python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: install build deps
+        run: |
+          python -V
+
+          python -m pip install build
+
+      - name: build sdist
+        run: |
+          export PYYAML_FORCE_CYTHON=1  # we DO want to force Cythoning, at least until 6.0
+          export PYYAML_FORCE_LIBYAML=0  # we don't actually want to build the lib
+
+          python -m build .
+
+          # ensure exactly one artifact was produced
+          shopt -s nullglob
+          DISTFILES=(dist/*.tar.gz)
+          if [[ ${DISTFILES[@]} -ne 1 ]]; then
+            echo "unexpected content in dist dir: $(ls dist/*.tar.gz)"
+            exit 1
+          fi
+
+      - name: test sdist
+        run: |
+          # install some libyaml headers
+          # TODO: should we smoke test the sdist against the libyaml we built?
+          sudo apt update
+          sudo apt install libyaml-dev -y
+
+          # ensure Cython is not present so we use only what's in the sdist
+          python -m pip uninstall Cython -y || true
```

Comment:
This shouldn't matter because unless you disable pip PEP517 build env isolation, it'll be built in a separate clean virtualenv that does not have Cython unless you add it as a build dep via `pyproject.toml`